TAPAS/Matrix constraintsMap
TAPAS/Matrix posRobotMapCenter
TAPAS/Matrix globalMapCenter